core/shared/src/main/scala/cats/effect/Bracket.scala

```diff
@@ -40,7 +40,9 @@ trait Bracket[F[_], E] extends MonadError[F, E] {
    * @param use $useParam
    * @param release is the action that's supposed to release the
    *        allocated resource after `use` is done, by observing
-   *        and acting on its exit condition
+   *        and acting on its exit condition. Throwing inside
+   *        this function leads is an undefined behaviour since it's
```

Comment:
```suggestion
   *        this function leads to undefined behavior since it's
```

* Cleans up the double verb
* [nit] We usually use American spellings over British, so I also changed "behavior"
* [nit] got rid of "an".  The indefinite article looks weird there.